modules/cms/classes/Page.php

```diff
@@ -46,7 +46,7 @@ class Page extends CmsCompoundObject
      */
     public $rules = [
         'title' => 'required',
-        'url'   => ['required', 'regex:/^\/[a-z0-9\/\:_\-\*\[\]\+\?\|\.\^\\\$]*$/i']
+        'url' => ['required', 'regex:/^\/[a-z0-9\/\:_\-\*\[\]\+\?\|\.\^\\\$]*$/i']
```

Comment:
Why has this spacing been adjusted?

Reply:
It is auto-format in PhpStorm. If the OctoberCMS team has xml code-format configuration file for PhpStorm - can easily apply it. Or make a settings correction, just in case - have one of the latest PhpStorm version 2017.3 and it was set by default.

Reply:
There's https://github.com/octobercms/october/blob/master/.editorconfig, but otherwise none of the core team uses PHPStorm so you'd have to change the settings yourself.

Reply:
Ok, thank you.